ibis/backends/tests/test_numeric.py

```diff
@@ -429,7 +429,7 @@ def test_floating_mod(backend, alltypes, df):
         'smallint_col',
         'int_col',
         'bigint_col',
-        'float_col',
+        pytest.param('float_col', marks=pytest.mark.broken("polars", strict=False)),
```

Comment:
Can you give a `reason=...` here? Why disable `strict`?

Reply:
Yes